components/hackathon/common/HackathonWinners.tsx

```diff
@@ -0,0 +1,90 @@
+import { Flex, Grid, Icon, Image, Link, VStack } from "@chakra-ui/react";
+import { FiExternalLink } from "react-icons/fi";
+import { Heading, Text } from "tw-components";
+
+interface IWinner {
+  name: string;
+  image: string;
+  link: string;
+  position: string;
+}
+
+interface HackathonWinnersProps {
+  winners: IWinner[];
+}
+
+export const HackathonWinners: React.FC<HackathonWinnersProps> = ({
+  winners,
+}) => {
+  return (
+    <VStack
```

Comment:
Can we use a <Flex flexDir="column" /> with some gap instead of a VStack?

Reply:
a?

Reply:
lmao, Flex

Reply:
ah gotcha